docs/_includes/common/menu.html

```diff
@@ -82,15 +81,15 @@ <h3 id="menu-radio">Radio Menu</h3>
                     </span>
                 </button>
                 <div class="menu__items" role="menu">
-                    <div class="menu__item" role="menuitemradio" aria-checked="true">
+                    <div class="menu__item" role="menuitemradio" aria-checked="true" data-menuitemradio-name="sort">
```

Comment:
As folks look at our examples, do we want this `data-menuitemradio-name` pattern to be picked up by developers (copy pasta)?

Reply:
Nope. I've left it out of the displayed example code.

Reply:
True, but a folks will also (hopefully not) look at the source and see how we're doing things to support the checked state. Not a big deal at all, mind you, just thinking out loud here.